### PR TITLE
Add pinned icon to original post

### PR DESCRIPTION
**Pull request by abeddow91**
## What does this change?
Adds a pinned icon to the original post that has been pinned. To do this, this PR passes the pinned post id through to liveblock so that it can be compared against each block id. If it matches the pinned post id but isn't itself the pinned post, it should render a pinned post icon next to the time and date stamp. 

This PR also passes the blocks format through so that the background colour of the icon can be derived from its format.

NB - pinned posts currently only exist in DCR not AR so the `isOriginalPinnedPost` prop in liveBlockContainer is optional. Once pinned posts are implemented in AR, this can become required.

## Why?
When a journalist pins a post, the original post stays in its original position in the live blog feed. Adding a pinned icon to this original post signifies it is the same post that is pinned above. 

### Before
Screenshot 2022-03-22 at 16 35 36

### After
Screenshot 2022-03-22 at 15 48 20

### CODE REVIEW

**Comment by oliverlloyd** [comment]
We have an issue for this one!

https://github.com/guardian/dotcom-rendering/issues/4346

**Review by joecowton1 — APPROVED** [review]
Looks great! Nice work.

File: common-rendering/src/components/FirstPublished.tsx [hunk 1]
```diff
@@ -66,6 +73,23 @@ const FirstPublished = ({
 					publishedDate.getMinutes(),
 				)}`}
 			</span>
+			{isOriginalPinnedPost && (
+				<span
+					css={css`
+						width: 0.875rem;
```
Comment (alinaboghiu) [inline common-rendering/src/components/FirstPublished.tsx:79]: I wonder if these should be px? we seem to be using px elsewhere in this file.

Comment (abeddow91) [reply common-rendering/src/components/FirstPublished.tsx:79]: It's a good question, I tend to favour rem where we can for scaling but use Source's space (with px) for spacing (margin, padding etc). @oliverlloyd is there any hard and fast rules for this in DCR?

Comment (mxdvl) [reply common-rendering/src/components/FirstPublished.tsx:79]: Here’s a one-button fix, @abeddow91:

```suggestion
						width: 14px;
```

Comment (abeddow91) [reply common-rendering/src/components/FirstPublished.tsx:79]: Great, thanks @oliverlloyd and @mxdvl !

**Pull request merged**